src/elements/card/src/stories.tsx

```diff
@@ -60,6 +61,7 @@ export const VerticalCards = () => {
               description={content}
               linkHref="https://www.uswitch.com"
               linkText="read about cats"
+              tag="Credit Card"
```

Comment:
could this be an array, lets check with Rob

Reply:
I've checked, he said no